src/test/groovy/com/blackducksoftware/integration/hub/detect/testutils/DependencyGraphResourceTestUtil.groovy

```diff
@@ -10,46 +10,54 @@ import com.google.gson.GsonBuilder
 
 class DependencyGraphResourceTestUtil {
     public static void assertGraph(String expectedResourceFile, DependencyGraph actualGraph) {
-        DependencyGraphSummarizer summarizer = new DependencyGraphSummarizer(new Gson());
+        DependencyGraphSummarizer summarizer = new DependencyGraphSummarizer(new Gson())

```

Comment:
I thought we were not removing semicolons because it made groovy conversion harder?